backend/python/trading_env/lib/python3.11/site-packages/h11/tests/test_io.py

```
@@ -0,0 +1,572 @@
+    w = ContentLengthWriter(5)
+    dowrite(w, Data(data=b"123"))
+    with pytest.raises(LocalProtocolError):
+        dowrite(w, Data(data=b"456"))
+
+    w = ContentLengthWriter(5)
+    dowrite(w, Data(data=b"123"))
+    with pytest.raises(LocalProtocolError):
+        dowrite(w, EndOfMessage())
+
+    w = ContentLengthWriter(5)
+    dowrite(w, Data(data=b"123")) == b"123"
```

Ruff (0.8.2)

Pointless comparison. Did you mean to assign a value? Otherwise, prepend `assert` or remove it.

(B015)